src/main/java/org/folio/marc4ld/service/ld2marc/processing/combine/factory/AbstractDataFieldCombiner.java

```diff
@@ -64,8 +64,8 @@ private void putRepeatableFields(DataField dataField) {
   }
 
   private void putFieldIfPresent(char fieldTag, DataField dataField) {
-    Optional.ofNullable(dataField.getSubfield(fieldTag))
-      .ifPresent(newField -> combinedField.addSubfield(newField));
+    Optional.ofNullable(dataField.getSubfields(fieldTag))
+      .ifPresent(subfields -> subfields.forEach(combinedField::addSubfield));
```

Comment:
dataField.getSubfields is not going to return null.

So, we could simplify this as 
```
  dataField.getSubfields(fieldTag)
    .stream()
    .forEach(combinedField::addSubfield);
```

Reply:
Done